test/guip.jl

```diff
@@ -1,11 +1,11 @@
-using Bridge, FixedSizeArrays, Distributions
+using Bridge, StaticArrays, Distributions
 using Base.Test
 #import Bridge: b, σ, a, transitionprob
-const percentile = 2.3 
+const percentile = 100.3 
 
```

Comment:
good question. perhaps `srand(.)` is the way to go. maybe to be creative `srand(package_version)`.